x-pack/test/detection_engine_api_integration/basic/tests/index.ts

```diff
@@ -27,6 +27,5 @@ export default ({ loadTestFile }: FtrProviderContext): void => {
     loadTestFile(require.resolve('./query_signals'));
     loadTestFile(require.resolve('./open_close_signals'));
     loadTestFile(require.resolve('./import_timelines'));
-    loadTestFile(require.resolve('./update_rac_alerts'));
```

Comment:
See below comment

Reply:
I think we can drop it, because it's not used now.